spec/models/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Comment Model' do
+    let(:user) { User.create(name: 'otsama', photo: 'photo', bio: 'software developer', post_counter: 5) }
+    let(:post) { Post.create(title: 'test title', text: 'test text', comments_counter: 1, likes_counter: 1) }
+
+    it 'text should not be blank' do
+      comment = Comment.new(author_id: user.id, post_id: post.id, text: 'this is comment')
+      comment.text = nil
+      expect(comment).to_not be_valid
+    end
+
+    it 'post id must be integer' do
+      comment = Comment.new(author_id: user.id, post_id: post.id, text: 'this is comment')
+      comment.post_id = nil
+      expect(comment).to_not be_valid
+    end
+
+    it 'user id must be integer' do
+      comment = Comment.new(author_id: user.id, post_id: post.id, text: 'this is comment')
+      comment.author_id = nil
+      expect(comment).to_not be_valid
+    end
+  end
```

Comment:
- [ ] Great job working on the validations so far but I think it would be great if you fix the failing tests since there are three failing tests in comments model after running them I faced these error messages so kindly take a look at them and try to solve them 🤩🚀💪🏻


- I think because your author_id is not named like this in your DB it is user_id so you should name it the same so your tests will pass ✨🤩✅✅


![image](https://github.com/Othniel2471/blog_app/assets/58553711/bb8a6587-e941-4024-afac-f3d73e508a86)